waddrmgr/db.go

```diff
@@ -109,12 +128,13 @@ type dbBIP0044AccountRow struct {
 // dbAddressRow houses common information stored about an address in the
 // database.
 type dbAddressRow struct {
-	addrType   addressType
-	account    uint32
-	addTime    uint64
-	syncStatus syncStatus
-	used       bool
-	rawData    []byte // Varies based on address type field.
+	addrType     addressType
+	account      uint32
+	addTime      uint64
+	syncStatus   syncStatus
+	used         bool
+	watchingOnly bool
```

Comment:
I think this is unnecessary too.  We could use 0 as the encrypted privkey length in the raw data.